### ISSUE TITLE
Add except: ["single-declaration"] to declaration-block-trailing-semicolon

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#5159]
allow `a { background: orange }` when declaration-block-trailing-semicolon is set to `"always"`
by introducing an except option that accepts `"single-declaration"`
i.e. if the block has 2 or more declarations, the last declaration still needs a `;`

e.g.
```json
        "declaration-block-trailing-semicolon": [
          "always",
          { "ignore": ["single-declaration"] }
        ],
```

It's useful for declaration blocks that **won't ever be expanded**:
```css
body { -webkit-animation: bugfix infinite 1s; }

/* from and to blocks will never change */
@-webkit-keyframes bugfix { from { padding: 0 } to { padding: 0 } }

**Comment by jeddy3** [comment]
Thanks for the request and for clearly explaining the use case.

A `"ignore": ["single-declaration"]` secondary option sounds good to me.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to add a new option](https://stylelint.io/developer-guide/rules#add-an-option-to-a-rule) in the Developer guide.

**Comment by Mouvedia** [comment]
@jeddy3 The problem with this option is that the exception cannot be limited to single lines only.
I will let the implementor figure out that one.

**Comment by Mouvedia** [comment]
> Please consider contributing if you have time.

@jeddy3 It was simpler than I expected. PR oncoming.
I didn't tackle the multi/single line granularity.
